server/migrations/20170821170950-book category.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  up(queryInterface, Sequelize) {
+    return queryInterface.createTable('BookCategories', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      category: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true,
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
```

Comment:
You can use the DATEONLY method for your createdAt and updatedAt props if you have no use for the entire value DATE returns.